anime-relations.txt

```diff
@@ -154,9 +169,15 @@
 # Date A Live II -> Date A Live: Encore OVA
 - 19163|7864:11 -> 22961|8511:1
 
+# Deadman Wonderland -> ~ OVA
+- 6880|4765:13 -> 10418|6172:1
```

Comment:
The OVA doesn't take place after the TV series, so it shouldn't be the 13th episode.